app/javascript/onboarding/components/__tests__/FollowUsers.test.jsx

```diff
@@ -148,13 +148,13 @@ describe('FollowUsers', () => {
     expect(queryByText(/You're not following anyone/i)).toExist();
 
     // select all then test following count
-    const followAllSelector = await findByText(/Select all 3 people/i);
+    const followAllSelector = await findByText(/Select all 3/i);
 
     fireEvent.click(followAllSelector);
 
     expect(queryByText('Follow')).not.toExist();
     expect(queryAllByLabelText('Following')).toHaveLength(3);
-    expect(queryByText(/You're following 3 people \(everyone\)/i)).toExist();
+    expect(queryByText(/You're following everyone/i)).toExist();
```

Comment:
Aha, I have figured out the problem with the tests - in the component you are using the `locale` function for this test, but the i18n data does not get populated for these Jest unit tests, hence the failures (i think in the test environment you will get some thing like "missing ${key} for current locale")

Elsewhere in the app we just use the locale function in the tests as well (see e.g. `app/javascript/Search/__tests__/Search.test.jsx`), which just kinda tests that the error messages match, but won't surface if the translation key really doesn't exist in the locale files. I guess you could go with that and we could treat properly mocking locales in unit tests as a separate issue?

Reply:
This was it! Thanks for spotting this! 🎉